### ISSUE TITLE
build.bat dont work correctly

### ISSUE THREAD

**Issue by COD3BREAK3R** [kiliman/operator-mono-lig#155]
When I click the buidl.bat file it creates the build folder but empty.

I have the latest version of your project, downloaded yesterday
fonttools  4.16.1
Python 3.7.3
Node 10.16.0
Windows 10 LTSC 1809 x64

I am attaching a screenshot of the "original" folder with the sources (ttx files are generated by build.bat):

![Captura de pantalla (84)](https://user-images.githubusercontent.com/51484063/98261414-707dbd00-1f52-11eb-9d3d-a15d4d8dcbf2.png)

When executing build.bat it produces this output that appears in this txt file:

[testlog.txt](https://github.com/kiliman/operator-mono-lig/files/5495335/testlog.txt)

**Comment by kiliman** [comment]
Looks like you didn't run `npm install`

**Comment by COD3BREAK3R** [comment]
It is true, I had not executed the command correctly, sorry.

It happens that I did not understand that part of the instructions about the npm install command well, I had to deduce after several tests that it was necessary to go from the console to the folder where the project is, then run npm install and finally run build.bat.

I recomend updating that part of the readme file with clearer instructions, I propose the following:

Finally from the terminal of our system go to the folder where you downloaded the project, once inside we execute the command npm install.

When it finishes downloading all the dependencies we execute the build file that corresponds to our system from our system explorer. For Windows: build.bat and for Mac / Linux: build.sh

--------------------------------------------------------------------

The non-SSm versions only add the ligatures to the Book, Book Italic, Light and Light Italic versions, ignoring the Medium and Medium Italic versions. However, in an older version of your project add ligatures to the Medium and Medium Italic versions.

For some reason the www ligature doesn't work for me

![Captura de pantalla (93)](https://user-images.githubusercontent.com/51484063/98392251-c0798400-2025-11eb-8491-7048129bc982.png)
